gardener/etcd-events.yaml

```diff
@@ -74,7 +74,16 @@ spec:
               version: v1
               name: etcd-events
               namespace: garden
-            patch: |
+            patch: |-
               - op: add
                 path: /spec/template/spec/priorityClassName
                 value: gardener-kubernetes
+              - op: replace
+                path: /spec/template/spec/containers/0/volumeMounts/0/name
+                value: etcd-events
+              - op: replace
+                path: /spec/template/spec/containers/1/volumeMounts/1/name
+                value: etcd-events
+              - op: replace
+                path: /spec/volumeClaimTemplates/0/metadata/name
+                value: etcd-events
```

Comment:
Honestly I would prefer if we could just rename the volume. I feel like these /0/ positional stuff is just bound to break

Reply:
Besides we will never be able to get rid of it.

Reply:
we are planning on contributing a new option to the helmchart like `volumeNameOverride`. Then this commit can be dropped.

And before we just rename it, id rather work on HA vgarden first, which will porbably result in new volumes anyways

Reply:
okay 👍